code/modules/migrations/mysql/mysql_migration.dm

```diff
@@ -0,0 +1,63 @@
+/datum/migration/mysql
```

Comment:
[Please document your code to meet dmdoc requirements.](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)